src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,785 @@
+		}
+	
+		return ['symfonyversion' => \Symfony\Component\HttpKernel\Kernel::VERSION];
+	}	
+	
+	/**
+	 * @Route("/breadcrumbs")
+	 * @Method("GET")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Admin breadcrumbs
+	 *
+	 * @return Response symfony response object
+	 */
+	public function breadcrumbsAction()
+	{		 
+		if (!$this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+			throw new AccessDeniedException();
+		}
+		
+		$masterRequest = $this->get('request_stack')->getMasterRequest();
+		$requested_cid = $masterRequest->attributes->get('acid');
+		$caller = [];
+		$caller['_zkModule'] = $masterRequest->attributes->get('_zkModule');
+		$caller['_zkType'] = $masterRequest->attributes->get('_zkType');
+		$caller['_zkFunc'] = $masterRequest->attributes->get('_zkFunc');
+		$caller['info'] = \ModUtil::getInfoFromName($caller['_zkModule']);
+				
+		if($caller['_zkModule'] == 'ZikulaAdminModule'){
+			$cid = empty($requested_cid) ? $this->getVar('startcategory') : $requested_cid;			
+		}else{			
+			$cid = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getmodcategory', array('mid' => $caller['info']['id']));
+		}			
+		$caller['category'] = \ModUtil::apiFunc('ZikulaAdminModule', 'admin', 'getCategory', array('cid' => $cid));
+		
+		return ['caller' => $caller];
+	}
+	
+	/**
+	 * @Route("/developernotices")
+	 *
+	 * @Template()
+	 * @Theme("admin")
+	 *
+	 * Open the admin container
+	 *
+	 * @return Response symfony response object
+	 */
+	public function developernoticesAction()
+	{
+		if (!$this->hasPermission('ZikulaAdminModule::', '::', ACCESS_ADMIN)) {
+			throw new AccessDeniedException();
+		}		 
+
+		$modvars = \ModUtil::getVar('ZikulaThemeModule');
+		
+		$data = array();
+		$data['devmode'] = $this->get('kernel')->getEnvironment() === 'dev';
+		
+		if ($data['devmode'] == true) {
+				$data['cssjscombine'] = $modvars['cssjscombine'];
+		
+				if ($modvars['render_compile_check']) {
+					$data['render']['compile_check'] = array('state' => $modvars['render_compile_check'],
+							'title' => $this->__('Compile check'));
+				}
+				if ($modvars['render_force_compile']) {
+					$data['render']['force_compile'] = array('state' => $modvars['render_force_compile'],
+							'title' => $this->__('Force compile'));
+				}
+				if ($modvars['render_cache']) {
+					$data['render']['cache']         = array('state' => $modvars['render_cache'],
+							'title' => $this->__('Caching'));
+				}
+				if ($modvars['compile_check']) {
+					$data['theme']['compile_check']  = array('state' => $modvars['compile_check'],
+							'title' => $this->__('Compile check'));
+				}
+				if ($modvars['force_compile']) {
+					$data['theme']['force_compile']  = array('state' => $modvars['force_compile'],
+							'title' => $this->__('Force compile'));
+				}
+				if ($modvars['enablecache']) {
+					$data['theme']['cache']          = array('state' => $modvars['enablecache'],
+							'title' => $this->__('Caching'));
+				}
+		}
```

Comment:
you might also add a note to all this that it is *smarty* information and that *twig* is configured differently and not possible to change via UI.